src/cloud/social/provider.ts

```diff
@@ -58,6 +58,14 @@ interface SavedContact {
   description?: string;
 }
 
+var UserStatus = {
```

Comment:
Ah thanks for catching!! I worked off of some older code.

Hmm I hadn't considered moving this into uproxy since we've been keeping social providers separate until now. I agree that there should be a solution to copy/pasting code from uproxy into different social providers though (we have this code in github too).